seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-algorithm-questions.json

```diff
@@ -23,7 +23,7 @@
         "telephoneCheck(\"555-555-5555\");"
       ],
       "solutions": [
-        "var re = /^([+]?1[\\s]?)?((?:[(](?:[2-9]1[02-9]|[2-9][02-8][0-9])[)][\\s]?)|(?:(?:[2-9]1[02-9]|[2-9][02-8][0-9])[\\s.-]?)){1}([2-9]1[02-9]|[2-9][02-9]1|[2-9][02-9]{2}[\\s.-]?){1}([0-9]{4}){1}$/;\n\nfunction telephoneCheck(str) {\n  return re.test(str);\n}\n\ntelephoneCheck(\"555-555-5555\");"
+        "var re = /^([+]?1[\s]?)?((?:[(](?:[2-9]1[02-9]|[2-9][02-8][0-9])[)][\s]?)|(?:(?:[2-9]1[02-9]|[2-9][02-8][0-9])[\s.-]?)){1}([2-9]1[02-9]|[2-9][02-9]1|[2-9][02-9]{2}[\s.-]?){1}([0-9]{4}){1}$/;\n\nfunction telephoneCheck(str) {\n  return re.test(str);\n}\n\ntelephoneCheck("555-555-5555");"
```

Comment:
The extra backslash on an escaped regex is due to how the test structure handles the JSON file where the challenges and tests are stored. The escaped quotes at the end are necessary for similar reasons, I believe. Was this causing an issue on the challenge in any way?

Reply:
I was unaware of the extra escapes needed for JSON. Does this become necessary then for the `\n`s throughout?

Reply:
@matthew-t-smith yes it should apply to all escaped characters in regex or js.